openreview/tools.py

```diff
@@ -35,7 +35,7 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
     profile = get_profile(client, email)
 
     if not profile:
-        response = client.get_tildeusername(first, last, middle)
+        response = client.get_tildeusername(first, last, None)
```

Comment:
Do we not want users to have middle names?

Reply:
the idea here is to validate the name as if it's just firstname and lastname.

for example, if there is "Michael Spector", and I try to create "Michael Li Spector", it throws an error.

that being said, your question made me realize that this affects more than just validation. I need to do two calls for tilde IDs: one to validate first and last, and another to get the proper id for first, middle, and last.

Reply:
I wonder if we can do the validation doing client.get_profile(first, last) instead of get_tilderusername